index.js

```diff
@@ -74,6 +74,27 @@ var prepareContext = function(ctx) {
   }
 
   /**
+   * Remove the bogus context from any standalone sassdoc comments.
+   * (We detect these if the context starts more than one line after the
+   * sassdoc comment ends.)
+   */
+  ctx.data.forEach(function(item) {
+    if (!item.context || !item.context.line) {
+      return;
+    }
+    if (item.context.type === 'unknown') {
+      item.context.type = 'prose';
+      item.context.line.end = item.context.line.start;
+    }
+    if (item.context.line.start > item.commentRange.end + 1) {
+      item.context = {
+        type: 'prose',
+        line: item.commentRange
+      };
+    }
```

Comment:
This block breaks cases where the Sass block selector is more than one line, which causes a line between `commentRange.end` and `context.line.start`. E.g. `_highlight.scss` `Code Blocks` section:

```scss
// Code Blocks
/// @name Code Blocks
/// @group code
/// @access private
/// @requires {function} hljs-gradient
[data-sassdoc='hljs'],
[data-sassdoc-page='index'] pre {
  align-items: stretch;
  background-color: color('hljs-background');
  color: color('hljs-text');
  display: flex;
  position: relative;
  ...
```